src/transcription/transcription.ts

```diff
@@ -112,8 +115,40 @@ export class SaladCloudTranscriptionSdk {
         validRequest.transcriptionId,
       )
       const { data } = response
-      // Validate and return the response payload.
-      return TranscribeResponseSchema.parse(data)
+      // Validate the response payload.
+      const validResponse = TranscribeResponseSchema.parse(data)
+
+      // Throw an error if the response contains an error message.
+      if (validResponse.output?.error) {
+        const errorMessage = `Transcription job ${validResponse.id} failed due to: ${validResponse.output.error}`
+        throw new Error(errorMessage)
```

Comment:
We should make this a custom error type with the message and id as properties.

Reply:
Introduced custom errors